simple/sample/output/process/report.json

```diff
@@ -1,37 +1,37 @@
 {
   "status": "SUCCESS",
-  "startTime": "2023-10-24 13:20:25.134318",
-  "lastUpdate": "2023-10-24 13:20:33.068269",
+  "startTime": "2023-10-24 15:07:01.417813",
```

Comment:
shall we start to have a mode to not update the timestamp in this file?

Reply:
Yes, I'll work on this following this PR.